hibernate-core/src/main/java/org/hibernate/engine/spi/SharedSessionContractImplementor.java

```diff
@@ -118,6 +118,15 @@ default SharedSessionContractImplementor getSession() {
 		return this;
 	}
 
+	/**
+	 * A "token" that is unique to this Session.
+	 *
+	 * @return The token
+	 */
+	default Object getSessionToken() {
+		return this;
+	}
+
```

Comment:
If this is v6 PR, maybe we can drop `default` here for we don't need to maintain backward compatibility and child class can choose to override it to customize easily.